sklearn/linear_model/coordinate_descent.py

```diff
@@ -786,19 +786,19 @@ class Lasso(ElasticNet):
 
     Parameters
     ----------
-    alpha : float, optional
-        Constant that multiplies the L1 term. Defaults to 1.0.
+    alpha : float, default 1.0.
```

Comment:
I can merge this but it's a commit for not much if we're not consistent. Someone will revert it in a next PR if there is no best practice.

@amueller @ogrisel @GaelVaroquaux what's your take on this? Can we start to agree on something in this PR?